.github/workflows/publish_charm.yaml

```diff
@@ -55,6 +59,28 @@ jobs:
           discourse_api_key: ${{ secrets.DISCOURSE_API_KEY }}
           dry_run: true
           github_token: ${{ secrets.GITHUB_TOKEN }}
+  publish-docs:
+    if: ${{ github.event.inputs.destination-channel }} == 'latest/stable'
```

Comment:
The PR description says:
```
Rationale
Enable docs publishing whenever a charm is published to any channel
```
however, it is filtered here to just the channel latest/stable

Reply:
Indeed. Let me double check something